src/commands/makeemotelist.js

```diff
@@ -24,11 +24,19 @@ class Command {
     
     const emote = getEmote(content);
     const isPM = message.channel.isPrivate;
-    jsonfile.readFile(assetsPath + "\\emotes.json", function(err, obj) {
-        for (var prop in obj) {
-            emotesListed += "<tr>\n    <td>" + prop + "</td>\n    <td>" + emotes[prop] + "</td>\n    <td><img src=\"" + emotes[prop] + "\" alt=\""+prop+"\"></td>\n</tr>";
+    jsonfile.readFile(assetsPath + '\\emotes.json', function(err, obj) {
+        if(err){console.log(err);}
+        if(content.search(new RegExp('imageless', 'i')) > -1) {
```

Comment:
No need to do this via RegExp here, just use `content.indexOf('imageless') > -1`